lms/templates/dashboard/_dashboard_entitlement_actions.html

```diff
@@ -29,10 +29,10 @@
     <ul class="entitlement-actions-dropdown-list" aria-label="${_('Available Actions')}" role="menu">
       % if can_refund_entitlement:
         <li class="entitlement-actions-item" role="menuitem">
-          ## href, id, rel, and data-model-close-button-selector must be defined for compatibility with lms/static/js/leanModal.js
+          ## href, rel, and data-model-close-button-selector must be defined for compatibility with lms/static/js/leanModal.js
           ## data-dropdown-selector and data-dropdown-button-selector must be defined for compatibility with lms/static/js/dashboard/dropdown.js
           ## class="js-entitlement-action-unenroll" must be set for compatibility with lms/static/js/learner_dashboard/views/entitlement_unenrollment_view.js
-          <a href="#entitlement-unenrollment-modal" id="entitlement-action-unenroll-${dashboard_index}" class="entitlement-action js-entitlement-action-unenroll" rel="leanModal"
```

Comment:
I might not have been clear when we talked about this earlier, but ID is required for the unenroll links because they use a different codepath for entitlements than the other links.

ID is NOT required for the email settings links because you're using the same codepath as the other links (code from legacy.js), which adds an ID to the link via Javascript.

Reply:
oh sorry that's an embarrassing mistake. I meant to remove the email settings modal id, not the unenrollment modal id